considerations/sip-editors.md

```diff
@@ -27,6 +29,8 @@ Invite tree:
 
 **Rafael Cárdenas**: Rafael is a Staff Software Engineer at Hiro, focused on building and maintaining Hiro’s API infrastructure which includes the Stacks Blockchain API, Ordinals API, Token Metadata API and others, all of which power dozens of apps and wallets built on Stacks and Bitcoin. Rafael has also been an active member and contributor of the Stacks ecosystem since 2021, involved with the development of the core blockchain through the SIP process and participating as a reviewer at the Stacks Accelerator.
 
+**werner.btc**: Since 2021, I have been an active Stacks community member and helper. As an artist, tester, and jack of many traits, my contributions are rooted in a passion for supporting the Stacks ecosystem. I am dedicated to working towards a more equitable, open internet that is inherently safe through the adoption of Bitcoin technology.
```

Comment:
The rest of these are in third person, can you update this one to match, e.g. "I have been" -> "Werner has been"